modules/apps/segments/segments-web/src/main/resources/META-INF/resources/css/_criteria_sidebar.scss

```diff
@@ -80,10 +80,13 @@
 		}
 	}
 
-	.properties-list {
+	.sidebar-collapse-body {
```

Comment:
What about taking advantage and move `padding-left: 0` and `flex-grow: 1` to utility classes? 😄

Reply:
jejeje you are right